src/components/ProgramRecord/ProgramRecord.scss

```diff
@@ -23,15 +26,17 @@
         .program-record-header {
 
             .program-headings {
-                @extend .d-flex, .justify-content-between;
-                color: $primary-500;
+                display: flex !important;
+                justify-content: space-between !important;
+                color: var(--pgn-color-primary-500);
 
                 h4 {
                     color: inherit;
                 }
 
                 .program-title {
-                    @extend .d-flex, .flex-column;
+                    display: flex !important;
+                    flex-direction: column !important;
```

Comment:
Based on https://github.com/openedx/frontend-app-authn/pull/1436#discussion_r2079869474 moving away from `@extend` in this way is good as-is!